joomla/joomla-cms
# [4.0] Ensure language code is validated in the mail template

Pull Request for Issue #30957 .

### Summary of Changes
Ensure there is a valid language code for an application when saving.

### Testing Instructions
You can't follow the instructions in the issue anymore as another PR has disabled the ability to navigate directly to the edit form. However you can reproduce by editing the form directly (as below) - this screenshot also shows the expected error when you save an item with an invalid language code:

![image](https://user-images.githubusercontent.com/1986000/103452012-d83a5500-4cc2-11eb-9f5d-b4e57c58edd1.png)

### Actual result BEFORE applying this Pull Request
Result saves or gives database error


### Expected result AFTER applying this Pull Request
Error message as in the above screenshot


### Documentation Changes Required
None
